fiftyone/utils/qwen3_vl.py

```
@@ -403,3 +411,149 @@ def embed_all(self, args):
+        message to Qwen3-VL, which processes the full temporal context
+        and returns one embedding vector.
+
+        Args:
+            video_reader: an ``eta.core.video.FFmpegVideoReader``
+
+        Returns:
+            a 1D numpy array embedding
+        """
+        from PIL import Image as PILImage
+
+        total_frames = video_reader.total_frame_count
```

Ruff (0.15.1)

[error] Local variable `total_frames` is assigned to but never used

Remove assignment to unused variable `total_frames`

(F841)